client/webserver/site/src/js/markets.js

```diff
@@ -188,14 +192,61 @@ export default class MarketsPage extends BasePage {
     ws.registerRoute(epochOrderRoute, data => { this.handleEpochOrderRoute(data) })
     // Bind the wallet unlock form.
     bindOpenWallet(app, page.openForm, async () => { this.openFunc() })
-    // Create a wallet
+    // Create a wallet.
     this.walletForm = new NewWalletForm(app, page.walletForm, async () => { this.createWallet() })
-    // Main order form
+    // Main order form.
     bindForm(page.orderForm, page.submitBttn, async () => { this.stepSubmit() })
-    // Order verification form
+    // Order verification form.
     bindForm(page.verifyForm, page.vSubmit, async () => { this.submitOrder() })
-    // Cancel order form
+    // Cancel order form.
     bindForm(page.cancelForm, page.cancelSubmit, async () => { this.submitCancel() })
+    // Bind active orders list's sort events.
+    bind(page.liveTypeCol, 'click', () => setOrdersSortCol('type'))
+    bind(page.liveSideCol, 'click', () => setOrdersSortCol('side'))
+    bind(page.liveAgeCol, 'click', () => setOrdersSortCol('age'))
+    bind(page.liveRateCol, 'click', () => setOrdersSortCol('rate'))
+    bind(page.liveQtyCol, 'click', () => setOrdersSortCol('qty'))
+    bind(page.liveFilledCol, 'click', () => setOrdersSortCol('filled'))
+    bind(page.liveSettledCol, 'click', () => setOrdersSortCol('settled'))
+    bind(page.liveStatusCol, 'click', () => setOrdersSortCol('status'))
+
+    const setOrdersSortCol = (key) => {
+      // First unset header's current sorted col classes.
+      unsetOrdersSortColClasses()
+      // If already sorting by key change sort direction.
+      if (this.ordersSortKey === key) {
+        switch (this.ordersSortDirection) {
+          case 'dsc':
+            this.ordersSortDirection = 'asc'
+            break
+          case 'asc':
+            this.ordersSortDirection = 'dsc'
+            break
+        }
+      } else {
+        // Otherwise update sort key and set default direction to descending.
+        this.ordersSortKey = key
+        this.ordersSortDirection = 'dsc'
+      }
+      this.refreshActiveOrders()
+      // Set header's new sorted col classes.
+      setOrdersSortColClasses()
+    }
+
+    const strCamelCase = (str) => `${str.charAt(0).toUpperCase()}${str.slice(1)}`
+
+    const unsetOrdersSortColClasses = () => {
+      const camelCaseKey = strCamelCase(this.ordersSortKey)
+      page[`live${camelCaseKey}Col`].classList.remove(`sorted-${this.ordersSortDirection}`)
+    }
+
+    const setOrdersSortColClasses = () => {
+      const camelCaseKey = strCamelCase(this.ordersSortKey)
+      page[`live${camelCaseKey}Col`].classList.add(`sorted-${this.ordersSortDirection}`)
+    }
```

Comment:
Please check the sense of direction on these arrows. I'm seeing

![image](https://user-images.githubusercontent.com/6109680/106683486-37bda600-658a-11eb-8f33-33c9f5e19848.png)

To me, the up arrow usually indicates ascending, such that the values will rise as you move down the table. I've checked a couple of other tables on other sites and my assumption appears to jive.